index.html

```diff
@@ -13,11 +13,36 @@
     -->
 
 	<!-- Meta -->
-	<title>Dogecoin</title>
+	<title>Dogecoin - #dogecoin - #doge</title>
```

Comment:
I think we can keep the webiste universal and don't need to add hashtags here.

Reply:
fixed